.c3i/config_v2.yml

```diff
@@ -37,6 +37,7 @@ tasks:
     user_feedback:
       title: "Conan v2 pipeline (informative, not required for merge)"
```

Comment:
```suggestion
      title: "Conan v2 pipeline"
```

Reply:
It's optional by default, only for those which are listed in the conan_v2_ready_references.yml they mandatory. It's better keeping it explicit, removing the informative section may cause confusion.

Reply:
Sorry, yes, the `title` field needs to be updated.

Reply:
Yes I put it in the description with a link :)

I think it's the right time to be more pushy about supporting v2

Reply:
> I think it's the right time to be more pushy about supporting v2

Agreed. :)